docs/reference/filters.md

```diff
@@ -1560,7 +1561,7 @@ Memory usage examples:
 Parameters:
 
 * number of allowed requests per time period (int)
-* time period for requests being counted (time.Duration)
+* time period for requests being counted (duration)
```

Comment:
Needs checking, as it also accepts (undocumented) integer as number of _seconds_

Reply:
and the change should not drop the godoc name, but maybe link it: [time.Duration](https://godoc.org/time#ParseDuration)

Reply:
The idea was to define `duration` type once at the top and reference it from filter args but it is too early, see https://github.com/zalando/skipper/pull/1814#issuecomment-895053332